udata/settings.py

```diff
@@ -128,7 +130,7 @@ class Testing(object):
     DEBUG_TOOLBAR = False
     SERVER_NAME = 'localhost'
     DEFAULT_LANGUAGE = 'en'
-
+    ACTIVATE_TERRITORIES = True
```

Comment:
Due to the overhead and to the already long running test, we might activate territories only on necessary test cases